hi-ml-histopathology/src/histopathology/datamodules/panda_module.py

```diff
@@ -45,8 +45,7 @@ def get_splits(self) -> Tuple[PandaDataset, PandaDataset, PandaDataset]:
                                                 proportion_train=.8,
                                                 proportion_test=.1,
                                                 proportion_val=.1,
-                                                subject_column=dataset.TILE_ID_COLUMN,
-                                                group_column=dataset.SLIDE_ID_COLUMN)
```

Comment:
why are we deleting slide_id? this seems unelated to mypy

Reply:
This is the PANDA slides dataset, so here we're splitting the dataset by slide ID. `mypy` was correctly complaining that `PandaDataset` does not have a `TILE_ID_COLUMN`.